### ISSUE TITLE
Helpers and Third Party parts are not shown in the "Parts" list

### ISSUE THREAD

**Issue by app4soft** [leozide/leocad#941]
**Describe the bug**

"Helpers" and "Third Party" parts, which names starts with "`|`" (Vertical bar, U+007C), are not shown in "Parts" list.

But such parts are visible via "Prefernces > Categories > Unassigned" and its possible to create a custom category with such parts via "Preferences > Categories" (even assigned to newly created catgories such parts still are invisible in the parts list).

**To Reproduce**

1. Install latest LDraw Parts Library (both `complete.zip` and `ldrawunf.zip` from https://library.ldraw.org);
2. Try to find in "Parts" list parts with the names started with "`|`" symbol

**Expected behavior**

LeoCAD should support "Helpers" and "Third Party" parts to be shown in the "Parts" list.

If there is no categories "Helpers" and "Third Party" yet, then "Helpers" and "Third Party" parts should be shown in "All Parts" category.

**Screenshots**

![LeoCAD_helpers1](https://github.com/user-attachments/assets/5ae92eb3-f78d-436b-a65f-bd62070be012)

![LeoCAD_helpers2](https://github.com/user-attachments/assets/26baa994-f734-44b0-8825-0655efd6c9d8)

**Version (please complete the following information):**

- OS: Debian
- LeoCAD Version: continuous

**Crash information:**

None

**Additional context**

Its now allowed to include "Helpers" and "Third Party" parts to the official LDraw Parts Library.

Both these categories has parts names started with "`|`" symbol.

(also, "Helpers" are stored in `helpers/` subfolder)

**Comment by nathaneltitane** [comment]
similar to #394 ?

**Comment by app4soft** [comment]
No, #394 could be simple fixed with switching to "All Parts" in "Parts" list.

@nathaneltitane, Actual issue is about parts started with some Special Prefix Characters (as you mentioned `=` in https://github.com/leozide/leocad/issues/394#issuecomment-569435556):

Actually in LeoCAD, parts with `~` and `_` prefixes are visible in "Parts" list.

But parts with `|` and `=` are invisible in "Parts" list;

- https://www.ldraw.org/article/398.html#descprefix

> Special Prefix Characters
> Some programs rely on the parts.lst file rather than the LDRAW_ORG meta-statement. Special prefix characters are added to the start of the part description to prevent the parts being included in the list by these programs:
> 
> - The descriptions of all subparts (i.e. line 4 = '0 !LDRAW_ORG Subpart') must start with '~'
> - The description of all obsolete parts, maintained for backwards compatibility must start with '~'
> - The use of '~' for non "s\" part files is at the author's discretion, to hide mouldings that are not released independently. This is one case where merely reading the LDRAW_ORG line is insufficient for tools to know how to treat the file.
> - **The descriptions of all aliases (i.e. line 4 = '0 !LDRAW_ORG Part Alias') must start with '='**
> - **The descriptions of all third party parts must start with '|' or '~|' (as appropriate)**
> - The descriptions of all physical colour parts (i.e. line 4 = '0 !LDRAW_ORG Part Physical_Colour') should start with '_' (This usage is depreciated)

**Comment by app4soft** [comment]
@leozide, Addition on this issue:

1. Parts with `~_=|` prefixes are shown in "All Parts" list;
2. But, such parts are not shown in appropriate categories, even parts with `~_=|` has `!CATEGORY` command defined.
3. If create a new category with matching keywords, parts with `~_=|` appears in a such new category;
4. All parts wth `|` in the prefixes are recognized as "Decorated Parts", even there are parts not decorated at all.
